### ISSUE TITLE
Account Suspended

### ISSUE THREAD

**Issue by Glynyrd** [dalaranwow/dalaran-wow#4267]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Cannot log in after a few months of not being on

**Expected behaviour**: Tell us what should happen instead
I should be able to log in to my account

**Steps to reproduce the problem**:

1. Attempt to log in
2. 
3. 

**Include proofs for this behaviour**

I don't understand what this means - do I need to take a screen shot of it not letting me in?

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
The issuetracker is serverside and gamemechanic issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. Thank you. (Create a new account for this issue)

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)

**Comment by Glynyrd** [comment]
I need to create a new account so I can write an in game issue for this? Just asking for clarification

**Comment by Rushor** [comment]
yep, i cannot help since i'm not a gamemaster. 

Quick create a new account and write the ingameissue then about your old (main) account. Maybe add this bugtrackerissue id too in your ingamereport. Thank you :)
